.pre-commit-config.yaml

```diff
@@ -26,7 +26,6 @@ repos:
         args: [--fix]
         additional_dependencies:
           - eslint
-          - eslint-config-react-app
```

Comment:
We did this for a lot of other projects too, funny that you found the same solution!

Reply:
Life is to short to properly deal with ESLint problems :sweat_smile: